src/main/telemetry/ghst.c

```diff
@@ -112,10 +113,97 @@ void ghstFramePackTelemetry(sbuf_t *dst)
     sbufWriteU8(dst, 0x00);                     // tbd3
 }
 
+// GPS data, primary, positional data
+void ghstFrameGpsPrimaryTelemetry(sbuf_t *dst)
+{
+    // use sbufWrite since CRC does not include frame length
+    sbufWriteU8(dst, GHST_FRAME_GPS_PAYLOAD_SIZE + GHST_FRAME_LENGTH_CRC + GHST_FRAME_LENGTH_TYPE);
+    sbufWriteU8(dst, GHST_DL_GPS_PRIMARY);
+
+    sbufWriteU32(dst, gpsSol.llh.lat);
+    sbufWriteU32(dst, gpsSol.llh.lon);
+
+    int32_t altitudeCm = gpsSol.llh.altCm;      // gps Altitude (absolute)
+    if (!STATE(GPS_FIX)) {
+        altitudeCm = 0;
```

Comment:
What is the intent of setting the altitude to 0 (e.g. sea level) without a GPS fix?